apps/bond/components/BondingProducts/Bonding/TokenManagement/WsolDeposit.jsx

```diff
@@ -118,12 +121,33 @@ export const WsolDeposit = () => {
         </Form.Item>
 
         <Form.Item>
-          <Spin spinning={isEstimating} size="small">
-            <Flex vertical gap="small" className="mb-16">
-              <Text strong>ESTIMATED OUTPUT</Text>
-              <Text>{`${estimatedOutput} WSOL-OLAS LP`}</Text>
-            </Flex>
-          </Spin>
+          <Table
+            columns={[
+              {
+                title: 'Estimated output',
+                dataIndex: 'estimated',
+                width: '50%',
+              },
+              {
+                title: 'Bridged token amount',
```

Comment:
Please call it `Current LP balance`

Reply:
Then also change `Estimated amount` to `Estimated LP amount`, and remove LP from both numbers below